src/less/time-column.less

```diff
@@ -31,6 +31,19 @@
 .rbc-day-slot {
   position: relative;
 
+  .rbc-events-container {
+    border-right: 10px solid transparent;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    width: 100%;
```

Comment:
any particular reason for this set of styles to accomplish this? as opposed to like 

```css
position: absolute;
top: 0, bottom: 0: left: 0; right 10px;
```

i'm not saying the above is better/worse just curious

Reply:
Honestly don't remember, I think I added this a few months back. I'll look over it...